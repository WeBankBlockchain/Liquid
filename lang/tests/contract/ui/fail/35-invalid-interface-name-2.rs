use liquid_lang as liquid;

#[liquid::interface(name = "fn")]
mod foo {
    extern "liquid" {
        fn bar();
    }
}

fn main() {}