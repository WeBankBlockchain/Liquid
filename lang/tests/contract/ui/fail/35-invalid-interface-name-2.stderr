error: invalid interface name: `fn`
 --> $DIR/35-invalid-interface-name-2.rs:3:28
  |
3 | #[liquid::interface(name = "fn")]
  |                            ^^^^
